experimental/dds/tree2/src/test/feature-libraries/defaultChangeFamily.spec.ts

```diff
@@ -129,97 +123,47 @@ describe("DefaultEditBuilder", () => {
+		expectForest(forest, {
+			type: jsonObject.name,
+			fields: {
+				foo: [{ type: jsonNumber.name, value: 42 }],
+			},
+		});
 		assert.equal(deltas.length, 2);
 
-		builder.setValue(root, 44);
-		expectForest(forest, { type: jsonNumber.name, value: 44 });
+		fooEditor.delete(0, 1);
 		assert.equal(deltas.length, 3);
-	});
-
-	it("Allows repair data to flow in and out of the repair store", () => {
-		const { builder, repairStore, changes, forest } = initializeEditableForest({
-			type: jsonNumber.name,
-			value: 41,
-		});
-
-		builder.setValue(root, 42);
-		expectForest(forest, { type: jsonNumber.name, value: 42 });
-
-		const change = changes[0];
-		const inverse = family.rebaser.invert(change, false, repairStore);
-		builder.apply(inverse);
-		expectForest(forest, { type: jsonNumber.name, value: 41 });
-	});
-
-	describe("Node Edits", () => {
-		it("Can set the root node value", () => {
-			const { builder, forest } = initializeEditableForest({
-				type: jsonNumber.name,
-				value: 41,
-			});
-			builder.setValue(root, 42);
-			expectForest(forest, { type: jsonNumber.name, value: 42 });
+		fooEditor.insert(0, singleTextCursor({ type: jsonNumber.name, value: 43 }));
+		expectForest(forest, {
+			type: jsonObject.name,
+			fields: {
+				foo: [{ type: jsonNumber.name, value: 43 }],
+			},
 		});
+		assert.equal(deltas.length, 4);
 
-		it("Can set a child node value", () => {
-			const { builder, forest } = initializeEditableForest({
-				type: jsonObject.name,
-				fields: {
-					foo: [
-						{ type: jsonNumber.name, value: 0 },
-						{ type: jsonNumber.name, value: 1 },
-						{
-							type: jsonObject.name,
-							fields: {
-								foo: [
-									{ type: jsonNumber.name, value: 0 },
-									{ type: jsonNumber.name, value: 1 },
-									{ type: jsonNumber.name, value: 2 },
-									{ type: jsonNumber.name, value: 3 },
-									{ type: jsonNumber.name, value: 4 },
-									{ type: jsonNumber.name, value: 5 },
-								],
-							},
-						},
-					],
-				},
-			});
-			builder.setValue(root_foo2_foo5, 42);
-			const expected = {
-				type: jsonObject.name,
-				fields: {
-					foo: [
-						{ type: jsonNumber.name, value: 0 },
-						{ type: jsonNumber.name, value: 1 },
-						{
-							type: jsonObject.name,
-							fields: {
-								foo: [
-									{ type: jsonNumber.name, value: 0 },
-									{ type: jsonNumber.name, value: 1 },
-									{ type: jsonNumber.name, value: 2 },
-									{ type: jsonNumber.name, value: 3 },
-									{ type: jsonNumber.name, value: 4 },
-									{ type: jsonNumber.name, value: 42 },
-								],
-							},
-						},
-					],
-				},
-			};
-			expectForest(forest, expected);
+		fooEditor.delete(0, 1);
+		assert.equal(deltas.length, 5);
+		fooEditor.insert(0, singleTextCursor({ type: jsonNumber.name, value: 44 }));
+		expectForest(forest, {
+			type: jsonObject.name,
+			fields: {
+				foo: [{ type: jsonNumber.name, value: 44 }],
+			},
 		});
+		assert.equal(deltas.length, 6);
```

Comment:
I think checking that three changes leads to three delta is sufficent, so we don't need this section.